rust/Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN apt-get update && apt-get install -y cmake curl clang git pkg-config libssl-
 RUN cargo build --locked --release -p processor
 RUN cp target/release/processor /usr/local/bin
 RUN cargo build --locked --release -p post-processor
+RUN cargo build --locked --release -p metrics-helper
```

Comment:
you will also need to copy this into the final image like on line 31.

`COPY --from=builder /usr/local/bin/metrics-helper /usr/local/bin`

Also nit maybe `metrics-helper` --> ` indexer-metrics` or something more descriptive?

Reply:
lol i see i see good point